core/templates/vector.h

```diff
@@ -114,6 +115,26 @@ class Vector {
 		}
 		return span().find(p_val, p_from);
 	}
+
+	Size find_seq(const Vector<T> &p_vector, Size p_from = 0) const {
```

Comment:
The name `find_sequence` would be better than `find_seq`. This is not a super common function so it should not be shortened.